lasagne/updates.py

```diff
@@ -54,9 +54,6 @@ def get_or_compute_grads(loss_or_grads, params):
 def sgd(loss_or_grads, params, learning_rate):
     """Stochastic Gradient Descent (SGD) updates.
 
-    Generates update expressions of the form:
-    * ``param := param - learning_rate * gradient``
-
```

Comment:
> Is it possible to add comments that apply to all update functions?

Such comments could be put in the module docstring, but I don't know what that will render like in Sphinx.